src_c/rwobject.c

```diff
@@ -736,8 +738,50 @@ _rwops_from_pystr(PyObject *obj)
 #if PY3
             if (PyUnicode_Check(obj)) {
                 SDL_ClearError();
-                PyErr_SetString(PyExc_FileNotFoundError,
-                                "No such file or directory.");
+
+                if (os_module) {
+                    PyObject* cwd = PyObject_CallMethod(os_module, "getcwd",
+                                                        NULL);
+                    if (cwd == NULL) {
+                        PyErr_SetString(PyExc_FileNotFoundError, 
+                                        "No such file or directory.");
+                        return NULL;
+                    }
+
+                    PyObject* path = PyObject_GetAttrString(os_module, "path");
+                    if (path == NULL) {
+                        Py_DECREF(cwd);
+                        PyErr_SetString(PyExc_FileNotFoundError,
+                                        "No such file or directory.");
+                        return NULL;                        
+                    }
+
+                    PyObject* abs = PyObject_CallMethod(path, "isabs", "O", obj);
```

Comment:
The `abs` variable needs a new name, because `abs` is already a function in C stdlib

Reply:
Thanks, good catch.